prow/resultstore/writer/writer.go

```diff
@@ -50,8 +53,8 @@ var (
 	rpcRetryDuration = 5 * time.Minute
 )
 
-func uuidString() string {
-	return uuid.New().String()
+func token() string {
+	return base64.StdEncoding.EncodeToString([]byte(uuid.New().String()))
```

Comment:
Can you explain why this is necessary (either as comments for this function, or in your commit message)?

Also it's unclear to me why you are renaming `uuidString` to `token`. As this function is not auth-related, maybe a better name is `random_token`?

Reply:
Added comments quoting the proto definitions. I had originall used uuidString() for both the tokens and the invocation ID before we agreed it should be the same as the prow job name. One function can't be used for both since the resume tokens require base64, so I've renamed these to correspond to their specific use.